src/rgw/rgw_user.cc

```diff
@@ -1565,9 +1565,8 @@ int RGWUser::update(const DoutPrefixProvider *dpp, RGWUserAdminOpState& op_state
 
   RGWUserInfo *pold_info = (is_populated() ? &old_info : nullptr);
 
-  ret = user->store_info(dpp, y, RGWUserCtl::PutParams()
-			 .set_old_info(pold_info)
-			 .set_objv_tracker(&op_state.objv));
+  ret = user->store_info(dpp, y, false, pold_info);
+  op_state.objv = user->get_version_tracker();
```

Comment:
i missed this on the first pass;  what is the store-independent notion of a version tracker?

Reply:
RGWObjVersionTracker has nothing store related in it.  It's two obj_versions, which are a uint64_t and a string.  We can wrap it if you want, but I didn't see the point.